framework/doc/content/framework/reviewing.md

```diff
@@ -0,0 +1,74 @@
+# Reviewing Pull Requests
+
+MOOSE is a large project with large numbers of users and contributors.
+While MOOSE seeks the highest standards of software quality, it
+is important to recognize that requiring perfection in proposed
+code changes is not realistic, given the large amount of activity,
+versus the limited availability of reviewers and time.
+The guidelines outlined on this page are designed to guide
+reviewers in finding the right balance between efficiency
+and software quality.
+
+## Conduct
+
+- Be respectful.
+- Try to be helpful. The reviewer is not expected to implement changes
+  for the author, but the reviewer should try to ensure that
+  the author has some direction, particularly if the author is a new
+  or infrequent contributor.
+- Make it clear what are changes that you require, versus suggestions.
+  For example, use "I suggest" for suggestions and imperative language,
+  e.g., "Do this", for required changes.
+- If it would be helpful, provide the author with a link to the contributing
+  guidelines page.
+- It is acceptable to make commits to the contributor's feature branch.
+  Note that NQA-1 explicitly allows this, and an additional, independent
+  review is only required if the additional commit(s) are significant.
+- If the author refuses the requested changes, request an additional
+  reviewer (ideally a `CODEOWNER` for the affected lines) to make a
+  final decision on the changes.
+
```

Comment:
```suggestion
- Be a team player. If there is a section of the code that you are not familiar with,
  feel free to ask for another reviewer's opinion.

```

Reply:
maybe this isnt the most team player advice in the end. Maybe we should find something to say we should act as a team on this though. It's important.